File: 프로그래머스/unrated/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.js
function solution(my_string, index_list) {
    var answer = [];
    let str = my_string.split("")
     for(let i =0; i<index_list.length;i++){
       answer.push(str[index_list[i]])
     }
    return answer.join().replaceAll(',','');
}

File: 프로그래머스/unrated/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.java
class Solution {
    public String solution(String my_string, int[] index_list) {
        String answer = "";
        String[] str = my_string.split("");
        for(int i =0; i<index_list.length;i++){
            answer += str[index_list[i]];
        }
        return answer;
    }
}